src/scheduler/workers/unpublish/getMetaForList.ts

```diff
@@ -27,13 +27,15 @@ export function getMetaForList(list: ListWithCountryName): Meta | undefined {
 
   const { keyDates } = currentAnnualReview;
 
-  const endDate = startOfDay(parseISO(keyDates.unpublished.UNPUBLISH));
   const startDate = startOfDay(parseISO(keyDates.annualReview.START));
+  const endDate = startOfDay(parseISO(keyDates.unpublished.UNPUBLISH));
   const today = startOfToday();
+  const daysUntilUnpublish = differenceInDays(endDate, today);
 
   return {
     reference: jsonData.currentAnnualReview!.reference,
-    weeksUntilUnpublish: differenceInWeeks(endDate, today),
+    daysUntilUnpublish,
+    weeksUntilUnpublish: differenceInWeeks(endDate, startOfToday()),
```

Comment:
it looks like daysUntilUnpublish and weeksUntilUnpublish will always be keys on `Meta`. (i.e. this function ALWAYS returns an object with those props). Why are they optional on the type?

Reply:
weeksUntilUnpublish are actually not required in the day namespace so I've moved Meta and getMetaForList to their respective namespaces